### PR TITLE
Skip Babylon.js from GitHub test repos

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
It seems to run out of space while compiling, so we should skip it.

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import ge = require("./getErrors");
 import pu = require("./packageUtils");
 import git = require("./gitUtils");
-import type { GitResult, UserResult } from './gitUtils'
+import type { GitResult, UserResult \} from './gitUtils'
```
Comment (sandersn) [inline main.ts:4]: uh oh

**Review by sandersn — CHANGES_REQUESTED** [review]
Something 'escaped' a whole bunch of characters in the file by inserting \ – so much that I missed the actual change.

File: main.ts (hunk 1 above)
Comment (DanielRosenwasser) [reply main.ts:4]: the heck

```suggestion
import type { GitResult, UserResult } from './gitUtils'
```

**Comment by DanielRosenwasser** [comment]
I have no idea how that happened. I wonder if it's a vscode.dev bug or if I screwed up somehow?

**Pull request closed without merging**